app/views/partials/_header-global.html.erb

```diff
@@ -0,0 +1,6 @@
+<header class="global-header">
+  <h1>You vs. The Machine</h1>
+  <p>The age-old struggle ends now.
+  Choose from the list of the problems,
+  and we'll supply the hacks to defeat your machine.</p>
```

Comment:
Don't break this text onto next lines with `Enter`.  Put it all on one line and let your text editor wrap it.

Everyone has their text editor at different widths - let it auto-wrap which works well for everyone rather than trying to do it manually.